### PR TITLE
[SCB-2187]support dynamic key values as yaml source 

### PR DESCRIPTION
**Pull request by kaister3**

### CODE REVIEW

File: demo/demo-springmvc/springmvc-server/src/main/resources/microservice.yaml [hunk 1]
```diff
@@ -47,10 +47,11 @@ servicecomb:
   # can download config center from https://cse-bucket.obs.myhwclouds.com/LocalCSE/Local-CSE-1.0.0.zip to test dynamic config
   config:
     client:
-  #    serverUri: http://127.0.0.1:30113
+      serverUri: http://127.0.0.1:30115
```
Comment (liubao68) [inline demo/demo-springmvc/springmvc-server/src/main/resources/microservice.yaml:50]: This test is an auto-test in docker and no config center installed. So you can not submit this test code.  
After you finished your local test, you can add test cases without config center. For example, add yaml files to `microservice.yaml`

```
mytest:
  yaml: |
      hostName: wyk
```

File: dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java [hunk 2]
```diff
@@ -17,13 +17,21 @@
 
 package org.apache.servicecomb.config.client;
 
-import java.util.List;
+import com.google.common.base.Joiner;
+import com.netflix.config.ConcurrentCompositeConfiguration;
+
+import java.util.*;
+import java.io.*;
+import java.net.*;
```
Comment (liubao68) [inline dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java:25]: Using `ctrl+alt+o` to organize imports. Do not use imortp x.x.*;

File: dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java [hunk 3]
```diff
@@ -71,6 +81,40 @@ private ConfigCenterConfig() {
...
     finalConfig = config;
+    readFileSourceConfig();
+  }
+
+  @SuppressWarnings("unchecked")
+  public List<String> getFileSource() {
+    return (List<String>) finalConfig.getProperty(FILE_SOURCE);
+  }
+
+  private static void readFileSourceConfig() {
```
Comment (liubao68) [inline dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java:92]: This PR is not correct. We are not going to support config yaml files. But config yaml contents in config center. Using your test case for example. 

mytest.yaml is a key in config center, and the value of mytest.yaml is the file content.

Comment (kaister3) [reply dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterConfig.java:92]: roger that

**Pull request closed without merging**